portfolio/src/main/webapp/index.html

```diff
@@ -17,22 +17,35 @@ <h1>Sarah Adebabay's Portfolio</h1>
       <button onclick="addRandomGreeting()">Fun Fact</button>
       <div id="greeting-container"></div>
     </div>
-    
     <div id="content">
       <h1>Let's Connect</h1>
       <h2>Github</h2>
       <a href="https://github.com/sarah-adebabay">https://github.com/sarah-adebabay</a>
       <h2>LinkedIn</h2>
       <a href="linkedin.com/in/sarah-adebabay">linkedin.com/in/sarah-adebabay</a>
+      <h2>Hello</h2>
+      <p>Click here for a fun surprise!</p>
+      <button onclick="getName()">Get it!</button>
```

Comment:
Hey Sarah, could you help explain what this button should show?

Right now the JavaScript `getName()` function is calling `fetch('/data')` which will be your comments. I think you'd have to write up a second Java servlet to have it show something different. Happy to help you with that if that's the goal!

Reply:
Hey Annie! I think I got mixed up with this part. I just wanted it to display my name or some funny message to test for the comment feature. I was able to fix it in my latest commit